gulpfile.js

```diff
@@ -344,12 +426,18 @@ gulp.task('watch', ['build'], function watchTask() {
 // Set up a webserver for the static assets
 gulp.task('connect', function connectTask() {
   connect.server({
-    root: DEST_PATH,
+    root: DIST_PATH,
     livereload: false,
-    port: 9988
+    port: 8000
```

Comment:
hmm, yeah. I was thinking about it from the angle of making sure it works with the addon. I guess either way something isn't going to work. I guess we could make a separate task for each...